libs/usb/JaRuleWidgetPort.cpp

```diff
@@ -102,6 +102,8 @@ JaRuleWidgetPort::~JaRuleWidgetPort() {
                   "CancelTransfer()?";
     }
 
+    // Cancelling may take up to a second if the endpoint has stalled. I can't
```

Comment:
SPaG Canceling is the American English, which the rest of the code seems to use.